practice-one/src/styles/components/button.css

```diff
@@ -0,0 +1,10 @@
+.button {
+  background-color: var(--color-primary);
+  border: none;
+  border-radius: 100px;
+  width: 162px;
+  height: 45px;
+  color: var(--color-light);
+  font-family: var(--font-family-normal);
```

Comment:
If this font family is primarily used for the whole app, then set font family for `html` then
```
html, body {
  font-family: var(--font-family-primary); // Rename this into font-family-primary
}
```

Reply:
I understood, thanks very much